chart/README.md

```diff
@@ -541,14 +541,62 @@ Some example commands are shown below.
    sonatype/nexus-iq-server-ha --version <version>
    ```
 
+### HPA (Horizontal Pod Autoscaling)
+
+The HPA in Kubernetes automatically scales workload resources based on demand. It deploys more Pods
+for increased load and scales down if the load decreases. Implemented as a Kubernetes API resource and controller, it
+adjusts the target scale periodically based on observed metrics like CPU and memory utilization.
+HPA is disabled by default. If you want to enable it, you need to set the `hpa.enabled` parameter to `true`, and set
+the `hpa.resources.memory.enabled` parameter to `true` to enable memory-based autoscaling.
+
+   ```
+   --set hpa.enabled=true
+   --set hpa.resources.memory.enabled=true
+   ```
+
+Note that HPA requires metrics-server to be installed in the cluster. If you are using EKS, you can install it by
+running the following
```

Comment:
I don't think this is EKS specific, right?

Reply:
Commit: 8524b9b2cf40a5f1a905c82307b9a25907a549e5